internal/crypto/csr.go

```diff
@@ -56,7 +56,7 @@ func ParseCSR(csrPEM []byte) (*x509.CertificateRequest, error) {
 	return csr, nil
 }
 
-func (ca *CA) IssueRequestedClientCertificate(csr *x509.CertificateRequest, expiryDays int) ([]byte, error) {
+func (ca *CA) IssueRequestedClientCertificate(csr *x509.CertificateRequest, expirySeconds int) ([]byte, error) {
```

Comment:
Would it be worth document these details?
```
// IssueRequestedClientCertificate issues a client certificate based on the provided 
// Certificate Signing Request (CSR) and the desired expiration time in seconds.
```

Reply:
Agree, always worth it to document.